chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -739,17 +737,19 @@ where
             frame_system::CheckNonce::<Runtime>::from(nonce),
             frame_system::CheckWeight::<Runtime>::new(),
             pallet_transaction_payment::ChargeTransactionPayment::<Runtime>::from(tip),
-            frame_metadata_hash_extension::CheckMetadataHash::<Runtime>::new(false),
+            //cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim::<Runtime>::new(),
+            frame_metadata_hash_extension::CheckMetadataHash::new(true),
```

Comment:
I see polkadot has frame_system::`WeightReclaim`::<Runtime>::new() instead of `StorageWeightReclaim`

Reply:
We don't have any of those, I created a ticket to check which extensions to add: MD-860

Reply:
ty